scripts/configs/template.conf

```diff
@@ -33,10 +35,13 @@ SampleNDRange = # currently unsupported
 
 [Harness]
 Name = harness_stencil
-Args = -p 2 -s 1024 
+Args = -s 1024
+#partially moved to environment.conf
```

Comment:
with partially you mean one specifies the OpenCL platform and device in the new environment config? I'd argue to specify OpenCL specifics still here since they are pretty much tied to all the other exploration parameters e.g., `global0` etc